src/book/entities/Transaction.ts

```diff
@@ -121,18 +121,8 @@ export async function updateCache(
     queryKey: [...Transaction.CACHE_KEY, { name: 'start' }],
   });
 
-  entity.splits.forEach(split => {
-    queryClient.invalidateQueries({
-      queryKey: [...Split.CACHE_KEY, split.accountId],
-    });
-  });
-
-  queryClient.invalidateQueries({
-    queryKey: [...Split.CACHE_KEY, { aggregation: 'total' }],
-  });
-
   queryClient.invalidateQueries({
-    queryKey: [...Split.CACHE_KEY, { aggregation: 'monthly-total' }],
+    queryKey: [...Split.CACHE_KEY],
```

Comment:
Given that now we have data bounded by intervals and better aggregation, we can just purge all splits as retrieving this data is not so costly.